oauthenticator/auth0.py

```diff
@@ -54,6 +68,10 @@ def _auth0_subdomain_default(self):
                 % self.__class__.__name__
             )
 
+    @default("logout_redirect_url")
+    def _logout_redirect_url_default(self):
+        return os.getenv('LOGOUT_REDIRECT_URL', '')
```

Comment:
Should the default be `"https://%s.auth0.com/v2/logout" % self.auth0_subdomain` like the authorize url? Is there a reason to *not* use this?

Let's also use `OAUTH_LOGOUT_REDIRECT_URL` to be consistent with other env names.

Reply:
> Should the default be "https://%s.auth0.com/v2/logout" % self.auth0_subdomain like the authorize url? Is there a reason to not use this?

I think it should. I just wasn't sure if we wanted the logout endpoint to be enabled by default.